aerogear/keycloak-metrics-spi
# Upgrade gradle wrapper, use implementation instead of compile

## Motivation
#106 
- build works with gradle 7

## What
- gradle wrapper is upgraded
- changed "compile" to "implementation" in build.gradle

## Why
- "compile" is deprecated since gradle 4
- build didn't work on my machine and I wanted to fix that.

## How
- Changed build.gradle "compile" statements
- upgraded the gradle wrapper

## Verification Steps
1. Go to project root
2. Run "./gradlew jar"
3. Check if build was successful 

## Checklist:


## Progress

- [x] Change statements in build gradle
- [x] Upgrade gradle wrapper

## Additional info
Wanted to create a JIRA ticket, because I didn't find any similar existing one. Registration process didn't work. Wrote an email to the red hat customer service. So, this (quite small issue) does not have a ticket yet.
